packages/svelte/src/internal/client/runtime.js

```diff
@@ -880,12 +875,16 @@ export function is_signal(val) {
  * @returns {T}
  */
 export function getContext(key) {
-	const context_map = get_or_init_context_map();
+	if (current_component_context === null) {
+		e.lifecycle_outside_component('getContext');
```

Comment:
why not move this logic into `get_or_init_context_map`? i.e. you give that function the `'getContext'` string and it will then invoke the error correctly. Should minify a bit better

Reply:
yeah, fair. made that change